llvm/include/llvm/IR/DataLayout.h

```diff
@@ -135,19 +115,26 @@ class DataLayout {
   // FIXME: `unsigned char` truncates the value parsed by `parseSpecifier`.
   SmallVector<unsigned char, 8> LegalIntWidths;
 
+  // Primitive type specifier.
+  enum class PrimitiveSpecifier {
+    Integer = 'i',
+    Float = 'f',
+    Vector = 'v',
+    // TODO: Aggregates are not primitives. This should be separated.
```

Comment:
Rename PrimitiveSpecifier -> TypeSpecifier then?

Reply:
I was hoping to to eliminate this enum member in observable future, but I'll rename to TypeSpecifier for now.